client/human/HumanClientApp.cpp

```diff
@@ -501,12 +501,14 @@ void HumanClientApp::NewSinglePlayerGame(bool quickstart) {
         else
             human_player_setup_data.m_empire_color = GG::CLR_GREEN;
 
+        const SpeciesManager& sm = GetSpeciesManager();
         human_player_setup_data.m_starting_species_name = GetOptionsDB().Get<std::string>("GameSetup.starting-species");
         if (human_player_setup_data.m_starting_species_name == "1")
             human_player_setup_data.m_starting_species_name = "SP_HUMAN";   // kludge / bug workaround for bug with options storage and retreival.  Empty-string options are stored, but read in as "true" boolean, and converted to string equal to "1"
+        else if (human_player_setup_data.m_starting_species_name == "RANDOM")
+            human_player_setup_data.m_starting_species_name = sm.RandomPlayableSpeciesName();
```

Comment:
maybe leave it "RANDOM" here, and instead handle the random choice during galaxy setup based on the seed value...